### ISSUE TITLE
Getting errors in obtaining hourly data

### ISSUE THREAD

**Issue by rajibshibly** [ropensci/weathercan#109]
## Expected Behavior
I have weathercan version 0.5 installed in R 4.04. I am trying to obtain all hourly data for the following station but having error message

Gander <- weather_dl(station_ids = 30346)

I get the following error message below:
Error: Problem with `mutate()` input `html`.
x could not find function "validate_key"
i Input `html` is `purrr::map(...)`.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

**Comment by steffilazerte** [comment]
Hi! Thanks for reporting the bug.

Hmm, I think there maybe something up with the internal caching done by the memoise package. What is your version of memoise? 

```
packageVersion("memoise")
```

What happens if you update memoise, restart R and try again?

```
install.packages("memoise")
```

If this fixes the problem, I'll increase the minimum version of memoise required.

**Comment by rajibshibly** [comment]
I updated all the packages and the error got fixed.
Thanks
Shibly
    On Monday, March 1, 2021, 12:26:32 PM GMT-3:30, Steffi LaZerte <notifications@github.com> wrote:  
 
 


Hi! Thanks for reporting the bug.

Hmm, I think there maybe something up with the internal caching done by the memoise package. What is your version of memoise?
packageVersion("memoise")

What happens if you update memoise, restart R and try again?
install.packages("memoise")

If this fixes the problem, I'll increase the minimum version of memoise required.
